src/List/style.css

```diff
@@ -0,0 +1,48 @@
+@import 'antd/lib/style/default.css';
+@import 'antd/lib/list/style/index.css';
+@import '../Empty/style.css';
+@import 'antd/lib/spin/style/index.css';
+@import 'antd/lib/pagination/style/index.css';
```

Comment:
pagination import needs to be our style

Reply:
my bad, initially i couldnt find the css file before merging with develop's latest commits